CHANGELOG.rst

```diff
@@ -9,7 +9,7 @@
 added
 =====
 
--  `SpatialSeries` and `DynamicTraceSegment` [:pull:`699`]
+-  ``SpatialSeries`` and ``DynamicTraceSegment`` [:pull:`699`]
```

Comment:
have you tried `~weldx.core.SpatialSeries` ?
Maybe it works if you add it to the user API documentation in `weldx/__init__.py`

Reply:
I think because of the user API some elements (our main classes) appear two times in the sphinx tree, e.g. once as `weldx.time.Time` and once as `weldx.Time` (user API)
In these cases the references have to be explicit or sphinx might throw an error (because it doesn't know which page to link to). But that should be mostly on the API documentation site

For the changelog I am nur sure what the sphinx context is for resolving these (or if there is any difference)